### ISSUE TITLE
use gui to add multiple floats

### ISSUE THREAD

**Issue by dempsey-CMAR** [dankelley/mooring#48]
Hello,

As discussed last week, I am interested in using the app to add multiple floats (sensors) to a wire moored in 2 - 100 m water depth.

**Comment by dankelley** [comment]
This is not too hard but (as you know) I'll be a bit buried in other things for a couple of weeks.

**Comment by dankelley** [comment]
`app2()` now handles an anchor-instrument-float assemblage.  I think the interface is pretty space-hungry and awkward, and I may try to tighten it up.  Adding more elements will be even worse, I think, and so I may not try that.  

The main thing is that the code generated by this app is a good starting point for complex moorings, and there is a vignette for a full-fledged mooring, so I'm not sure there is sufficient cost: benefit analysis to creating a more general multiple-element system.

Basically, it's more work to click and slide a lot of GUI elements than it is to just copy-paste and then edit code.
